src/main/java/LevelController.java

```diff
@@ -0,0 +1,88 @@
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.canvas.Canvas;
+import javafx.scene.layout.StackPane;
+
+import java.io.File;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.ResourceBundle;
+
+/**
+ * Created by Jim on 9/5/2015.
+ */
+public class LevelController implements Initializable {
+
+    /**
+     * The StackPane the level is drawn on.
+     */
+    @FXML
+    private StackPane root;
+
+    /**
+     * The canvas that is in the StackPane.
+     */
+    @FXML
+    private Canvas canvas;
+
+    /**
+     * The list of maps that the user is about to play.
+     */
+    private ArrayList<String> maps;
+
+    /**
+     * The current level the user is playing.
+     */
+    private int currLvl;
+
+    /**
+     * The init function.
+     * @param location The URL
+     * @param resources The ResourceBundle.
+     */
+    @Override
+    public final void initialize(final URL location, final ResourceBundle resources) {
+        maps = new ArrayList<>();
+        findMaps();
+
+        if (maps.size() > 0) {
+            currLvl = 0;
+            createLvl();
+        } else {
+            System.out.println("No maps found!");
+        }
+
+    }
+
+    /**
+     * This function scans the resources folder for maps.
+     */
+    private void findMaps() {
+        File folder = new File("c:/users/jim/workspace/bubblebobble/src/main/resources");
```

Comment:
This seems to be a really bad idea; my name is not Jim, and I do not use Windows.